modules/nf-core/modules/samtools/view/main.nf

```diff
@@ -1,5 +1,5 @@
 process SAMTOOLS_VIEW {
-    tag "$meta.id"
+   // tag "$meta.id" // This causes:  java.lang.NullPointerException: Cannot get property 'id' on null object. -- Check script '/home/owacker/git/sarek/./workflows/../subworkflows/nf-core/../../modules/nf-core/modules/samtools/view/main.nf' at line: 2 or see '.nextflow.log' file for more details
```

Comment:
need to fix this then, there is something funky going on, meta.id must always be there